src/main/java/coydir/ui/PersonInfo.java

```diff
@@ -0,0 +1,71 @@
+package coydir.ui;
+
+import java.util.Comparator;
+
+import coydir.model.person.Person;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+
+/**
+ * An UI component that displays detailed information of a {@code Person}.
+ */
+public class PersonInfo extends UiPart<Region> {
+    private static final String FXML = "PersonInfo.fxml";
+
+    private Person person;
+
+    @FXML
+    private HBox personInfo;
+    @FXML
+    private Label name;
+    @FXML
+    private Label employeeId;
+    @FXML
+    private Label position;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label address;
+    @FXML
+    private Label email;
+    @FXML
+    private FlowPane tags;
+
+    /**
+     * Creates a {@code PersonInfo} to display the {@code Person} particulars.
+     */
+    public PersonInfo(Person person) {
```

Comment:
I don't really get it, but this constructor looks to be the same as the `PersonInfo::update` method below. Why not call `update` in the constructor instead of duplicating the function?

Reply:
resolved